compiler/circle-mpqsolver/src/CircleMPSolver.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Methods/BisectionSolver.h"
+
+#include <arser/arser.h>
+#include <vconone/vconone.h>
+
+#include <iostream>
+#include <iomanip>
+#include <chrono>
+
+void print_version(void)
+{
+  std::cout << "circle-quantizer version " << vconone::get_string() << std::endl;
+  std::cout << vconone::get_copyright() << std::endl;
+}
+
+int entry(int argc, char **argv)
+{
+  arser::Arser arser("circle-mpsolver provides circle_model mixed precision quantization");
+
+  arser::Helper::add_version(arser, print_version);
+  arser::Helper::add_verbose(arser);
+
+  arser.add_argument("--data").required(true).help(".h5 file with test data");
+  arser.add_argument("--accuracy_ratio")
+    .type(arser::DataType::FLOAT)
+    .default_value<float>(0.5f)
+    .help("accuracy ratio");
+
+  //  arser.add_argument("--method")
+  //    .type(arser::DataType::STR_VEC)
+  //    .help("Currently only bisection is supported");
```

Comment:
You may receive arguments by command line, e.g.,

```
circle-mpqsolver --data <data> --bisection <option_1> ... <option_n> --input_model <model> --output_model <model>
```

Or you may receive configuration from json file (perhaps compatible with quantization configuration file). This may be  useful if circle-mpqsolver allows user's tuning, e.g., enforcing q16 to specific layers.

Reply:
Ok.

Reply:
For now introduced int16_at_input option for --bisection, so we can manually set where most sensitive to quantization layers are. But json option seems like quite usable, i need some more information, what values it should include.